src/main/resources/org/craftercms/studio/api/v2/dal/ItemDAO.xml

```diff
@@ -49,6 +46,14 @@
         <result property="ignoredAsInt" column="ignored" />
         <result property="childrenCount" column="children_count" />
         <result property="previousPath" column="previous_path" />
+        <association property="modifier" column="modifier"
```

Comment:
@sumerjabri please let me know your thoughts about these changes.
Each of these associations will translate to a User table query (although MyBatis is smart about it and will not request multiple times the same item).
The advantage is that it will reduce the number of joins in several queries and it will make it much easier to maintain.

Reply:
I guess there might be some performance impact if having many items submitted by many different authors, otherwise it should remain pretty similar to what we have today.

Reply:
I think this is a major benefit. We may end up tuning the DB schema if this proves too expensive.